blots/scroll.ts

```diff
@@ -164,10 +164,10 @@ class Scroll extends ScrollBlot {
     const last = lines.pop();
     let after;
     if (last != null) {
-      after = child.split(offset + first.delta.length());
+      after = child.split(offset + first.delta.length(), true);
```

Comment:
We neet to enable `force` to make sure a newline char is created even `offset + first.delta.length()` is at the edge of `child`.